src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_panel.tsx

```diff
@@ -184,6 +184,7 @@ export const WorkspacePermissionSettingPanel = ({
       {isEditing && (
         <EuiSmallButton
           fullWidth={false}
+          style={{ maxWidth: 160 }}
```

Comment:
Instead of giving a fixed width, I suggest do this:
```typescript
      {isEditing && (
        <EuiCompressedFormRow fullWidth>
          <EuiSmallButton
            fullWidth={false}
            onClick={handleAddNewOne}
            data-test-subj={`workspaceForm-permissionSettingPanel-addNew`}
            color="primary"
            iconType="plusInCircle"
          >
            {i18n.translate('workspace.form.permissionSettingPanel.addCollaborator', {
              defaultMessage: 'Add collaborator',
            })}
          </EuiSmallButton>
        </EuiCompressedFormRow>
      )}
```

Reply:
updated